components/Home/Sponsors/sponsors.jsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+import sponsorData from "../../../app/data/home/sponsors.json";
```

Comment:
Follow Next.js file path conventions by avoiding the use of `../../../` in your imports.